core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -353,6 +354,7 @@ public void taskStarted(Executor executor, Queue.Task task) {
 
     @Override
     public void taskCompleted(Executor executor, Queue.Task task, long durationMS) {
+        LOGGER.log(Level.INFO, "Completed {0} on {1}", new Object[] {task.getFullDisplayName(), executor.getOwner().getDisplayName()});
```

Comment:
We only fairly recently turned down the logging level of build finishes (#4345? I thought there was something else more recent, but cannot find it), this looks like it might effectively restore that (and be even more verbose, given Pipelines can have many tasks IIUC).
```suggestion
        LOGGER.log(Level.FINE, "Completed {0} on {1}", new Object[] {task.getFullDisplayName(), executor.getOwner().getDisplayName()});
```